src/main/java/stsjorbsmod/cards/cull/WastingEssence.java

```diff
@@ -22,8 +22,8 @@
 
     private static final int COST = 3;
     private static final int NUMBER_OF_CURSES = 5;
-    private static final int DAMAGE_PER_CURSE = 20;
-    private static final int UPGRADE_DAMAGE_PER_CURSE = 10;
+    private static final int DAMAGE_PER_CURSE = 12;
+    private static final int UPGRADE_DAMAGE_PER_CURSE = 4;
 
     public WastingEssence() {
```

Comment:
In addition to the damage nerf, this was renamed to `Wasting Form` (we avoid renaming classes/IDs for Wanderer cards since it's a "released" character to avoid breaking save files, but I think it's alright to do it for CULL cards at this point in its development)